sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/components/CreateNotificationPresenter.groovy

```diff
@@ -21,10 +22,11 @@ class CreateNotificationPresenter implements CreateNotificationOutput, FetchSubs
     }
 
     @Override
-    void createdNotifications(Map<Subscriber, String> notificationPerSubscriber) {
-        notificationPerSubscriber.each {
-            this.notificationPerSubscriber.put(it.key, it.value)
-        }
+    void createdNotifications(List<NotificationContent> notifications) {
+      //TODO
```

Comment:
Not sure if this is to be tackled in another branch/by somebody else

Reply:
maybe @Steffengreiner can comment on that?:)

Reply:
I think this will be where the dependency manager will get the List of notificationContent needed for the email generation

Reply:
ok, but is this going to be implemented with your PR?

Reply:
Yes I'll adapt the fill PR #17 as soon as this is merged to use this list 👍